src/calendar-list/index.tsx

```diff
@@ -343,12 +348,14 @@ class CalendarList extends Component<Props, State> {
   }
 
   render() {
-    const {style, pastScrollRange, futureScrollRange, horizontal, showScrollIndicator} = this.props;
+    const {style, pastScrollRange, futureScrollRange, horizontal, showScrollIndicator, numberOfColumn} = this.props;
 
     return (
       <View style={this.style.flatListContainer}>
         <FlatList
+          key={numberOfColumn}
```

Comment:
Why do you need to pass 'key' here? Do you need to reconstruct the FlatList when the 'numberOfColums' change?